chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertificationResultManager.java

```diff
@@ -336,15 +348,12 @@ public int update(final CertifiedProductSearchDetails existingListing,
     private Boolean haveMacraMeasuresChanged(List<MacraMeasure> orig, List<MacraMeasure> updated) {
         if (orig != null && updated != null) {
             return !orig.equals(updated);
-        } else if (orig != null && updated == null) {
-            return true;
-        } else if (orig == null && updated != null) {
```

Comment:
Why are these other branches removed?

Reply:
It was suggested by Checkstyle that they should be removed.

Reply:
Then checkstyle is wrong. If the original thing is null, and the updated thing is not null, then we want this method to tell us that the measures have changed, so it needs to return true. In that case, with the new code, we do not return from line 350, because in 349 at least one of the things is null, so we pass that check. That takes us to the else, which, in the first clause, checks that `orig != null`, but `orig == null`, so it short-circuits the conditional and returns `false`, which is incorrect

Reply:
I'll revert the code....
Just to show I'm not crazy!  
![image](https://user-images.githubusercontent.com/24817202/95217812-9da74600-07c1-11eb-96cc-96659dd01c5a.png)
Then when I click on the "Simplify return statement" link, it automatically refactored to:
![image](https://user-images.githubusercontent.com/24817202/95218013-d3e4c580-07c1-11eb-9ad7-56cb13da7777.png)

Reply:
Never said you were crazy! Just walking through the logic...